api/scripts/wazuh-apid.py

```diff
@@ -6,13 +6,12 @@
 import logging
 import os
 import sys
-import yaml
 import ssl
 
 import connexion
 from flask_cors import CORS
 
-from api import encoder, configuration
+from api import encoder, configuration, __path__ as api_path
```

Comment:
Just a suggestion to check if the following links fit better what we want achieve here:
* https://stackoverflow.com/questions/247770/how-to-retrieve-a-modules-path
* https://stackoverflow.com/questions/5183601/how-to-get-the-current-running-module-path-name/8789689

Reply:
I think the `__path__` variable fits our needs. From the [docs](https://docs.python.org/3.7/tutorial/modules.html#packages-in-multiple-directories):
> Packages support one more special attribute, `__path__`. This is initialized to be a list containing the name of the directory holding the package’s` __init__.py` before the code in that file is executed. This variable can be modified; doing so affects future searches for modules and subpackages contained in the package.